x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/engines_list.tsx

```diff
@@ -35,106 +37,112 @@ export const EnginesList: React.FC = () => {
   }, [meta.from, meta.size, searchQuery]);
 
   return (
-    <EnterpriseSearchContentPageTemplate
-      pageChrome={[
-        i18n.translate('xpack.enterpriseSearch.content.engines.breadcrumb', {
-          defaultMessage: 'Engines',
-        }),
-      ]}
-      pageHeader={{
-        pageTitle: i18n.translate('xpack.enterpriseSearch.content.engines.title', {
-          defaultMessage: 'Engines',
-        }),
-        rightSideItems: [
-          <EuiButton
-            fill
-            iconType="plusInCircle"
-            data-test-subj="enterpriseSearchContentEnginesCreationButton"
-            href={'TODO'}
-          >
-            {i18n.translate('xpack.enterpriseSearch.content.engines.createEngineButtonLabel', {
-              defaultMessage: 'Create engine',
-            })}
-          </EuiButton>,
-        ],
-      }}
-      pageViewTelemetry="Engines"
-      isLoading={false}
-    >
-      <EuiText>
-        {i18n.translate('xpack.enterpriseSearch.content.engines.description', {
-          defaultMessage:
-            'Engines allow you to query indexed data with a complete set of relevance, analytics and personalization tools. To learn more about how engines work in Enterprise search ',
-        })}
-
-        <EuiLink data-test-subj="documentationLink" href="TODO" target="_blank">
-          {i18n.translate('xpack.enterpriseSearch.content.engines.documentation', {
-            defaultMessage: 'explore our Engines documentation',
-          })}
-        </EuiLink>
-      </EuiText>
-      <EuiSpacer />
-      <div>
-        <EuiFieldSearch
-          value={searchQuery}
-          placeholder={i18n.translate('xpack.enterpriseSearch.content.engines.searchPlaceholder', {
-            defaultMessage: 'Search engines',
-          })}
-          aria-label={i18n.translate('xpack.enterpriseSearch.content.engines.searchBar.ariaLabel', {
-            defaultMessage: 'Search engines',
+    <>
+      <DeleteEngineModal />
+      <EnterpriseSearchContentPageTemplate
+        pageChrome={[
+          i18n.translate('xpack.enterpriseSearch.content.engines.breadcrumb', {
+            defaultMessage: 'Engines',
+          }),
+        ]}
+        pageHeader={{
+          pageTitle: i18n.translate('xpack.enterpriseSearch.content.engines.title', {
+            defaultMessage: 'Engines',
+          }),
+          rightSideItems: [
+            <EuiButton
+              fill
+              iconType="plusInCircle"
+              data-test-subj="enterpriseSearchContentEnginesCreationButton"
+              href={'TODO'}
+            />,
+          ],
+        }}
+        pageViewTelemetry="Engines"
+        isLoading={false}
+      >
+        <EuiText>
+          {i18n.translate('xpack.enterpriseSearch.content.engines.description', {
```

Comment:
nitpick: Rather than breaking up the text into multiple separate translation strings, which will make for very poor translations, we should be using `<FormattedMessage>` to include them in the same translation. For an example of how you can do this:
https://github.com/elastic/kibana/blob/61b8ef43639f2705b8b7e5d0941ea811004ba8bd/x-pack/plugins/enterprise_search/public/applications/shared/error_state/error_state_prompt.tsx#L61